src/controllers/expense.controller.js

```diff
@@ -0,0 +1,74 @@
+const statusCodes = require('./../utils/statusCodes');
+const expenseService = require('./../services/expense.service');
+const userService = require('./../services/user.service');
+
+const getAll = (req, res) => {
+  const expenses = expenseService.getAll(req.query);
+
+  res.status(statusCodes['OK']).send(expenses);
```

Comment:
```suggestion
  res.status(statusCodes.OK).send(expenses);
```
Use square brackets when you need to access with a dynamic key, in this case, you can access with a dot, fix in all places